project/MiMa.scala

```diff
@@ -683,7 +683,16 @@ object MiMa extends AutoPlugin {
 
         // #21894 Programmatic configuration of the ActorSystem
         ProblemFilters.exclude[DirectMissingMethodProblem]("akka.actor.ActorSystemImpl.this")
-      ) ++ bcIssuesBetween24and25)
+
+      ) ++ bcIssuesBetween24and25),
+      "2.4.16" -> Seq(
+        // #22035 Make it possible to use anything as the key in a map
+        FilterAnyProblemStartingWith("akka.cluster.ddata.protobuf.msg.ReplicatedDataMessages"),
+        FilterAnyProblemStartingWith("akka.cluster.ddata.ORMap"),
+        FilterAnyProblemStartingWith("akka.cluster.ddata.LWWMap"),
+        FilterAnyProblemStartingWith("akka.cluster.ddata.PNCounterMap"),
+        FilterAnyProblemStartingWith("akka.cluster.ddata.ORMultiMap")
+      )
```

Comment:
sorry, I was wrong. Revert this and place these in the section `bcIssuesBetween24and25` instead.